src/main/java/baritone/process/FarmProcess.java

```diff
@@ -166,8 +191,119 @@ private boolean isNetherWart(ItemStack stack) {
         return !stack.isEmpty() && stack.getItem().equals(Items.NETHER_WART);
     }
 
+    private boolean putDropsInChest(NonNullList invy) {
+        List<Slot> inv = ctx.player().openContainer.inventorySlots;
+        NonNullList<ItemStack> invx = invy;
+        for (int i = 0; i < invx.size(); i++) {
+            if (!invx.isEmpty() && PICKUP_DROPPED.contains(invx.get(i).getItem())) {
+                for (int j = 0; j < inv.size() - invx.size(); j++) {
+                    if (inv.get(j).getStack().isEmpty()) {
+                        ctx.playerController().windowClick(ctx.player().openContainer.windowId, i < 9 ? inv.size() - i / 9 * 9 - 9 + i % 9 : inv.size() - invx.size() + i - 9, 0, ClickType.QUICK_MOVE, ctx.player());
```

Comment:
BradyToday at 7:32 PM
what the fuck
what the fuck
 
BradyToday at 7:32 PM
what the fuck
what the fuck
who did this

Reply:
I know this doesn't look great but I always had NPE s or it didn't work to put the stuff in the chest

Reply:
I can explain what i did , and why i did it ,if you want

Reply:
ah your right ,this  is a leftover , when i didn't recognise ths: 

> that in the player inventory the hotbar is counted first

Reply:
i wil remove the unnecessary modulo operation when I am back home ,bc the fan of my laptop broke an hour ago , and its not fun to code with a 800 MHz Cpu

Reply:
Elenterius change works fine so it would be nice to , know if there still is a problem with this part of the code